src/sdk/client.ts

```diff
@@ -150,10 +150,10 @@ export const makeRequestFactory = (
   return createRequest;
 };
 
-const toQueryString = (query: Record<string, string>): string => {
+const toQueryString = (query: Record<string, string | number>): string => {
   const params = new URLSearchParams();
   for (const [key, value] of Object.entries(query)) {
-    params.set(key, value);
+    params.set(key, `${value}`);
```

Comment:
Is this needed? Tested it in the browser and it seems like it produces the same?
<img width="282" alt="image" src="https://github.com/vesselapi/integrations/assets/12945091/a592145e-769e-4baa-a4f1-8b25986864f1">

Reply:
the types on `param` set don't let me do it